R/get_wasserportal_master_data.R

```diff
@@ -0,0 +1,46 @@
+#' Helper function: Base Url of Berlin Wassersportal
+#'
+#' @return string with base url of Berlin Wasserportal
+#' @export
+
+wasserportal_base_url <- function() {
+  "https://wasserportal.berlin.de"
+}
+
+#' Wasserportal Berlin: get master data for stations
+#'
+#' @param station_id station_id
+#' @param url_wasserportal base url to Wasserportal berlin (default: 
+#' wasserportal_base_url())
+#' @return data frame with metadata for 
```

Comment:
for what?

Reply:
Thought of doing it @hsonne style and make it adaptable (in case the url changes). But may be that was too much...

Reply:
Absolutely, that's great. What I meant is:

> #' return data frame with metadata for ...

what for?